README.md

```diff
@@ -1,2 +1,33 @@
 # What am I Forging up
-A Discord bot used to monitor API usage statistics of Minecraft mods.
\ No newline at end of file
+A Discord bot used to monitor API usage statistics of Minecraft mods.
+
+### Quick Start
+To get WAIFU up and running as soon as possible you will need the following:
+* A Docker Installation
+* A CurseForge API token (Obtainable from https://console.curseforge.com/)
+* A Discord bot token (Obtainable from https://discord.com/developers/applications)
+
+If you have all of these simply clone the repo and open a terminal window in it, 
+then follow these steps:
+
+1) Create a "run" directory in the root of the repo, copy the contents of run.template
+into it, and fill out the secrets db_user.txt and db_pass.txt to create the root login
```

Comment:
And of course only after this is accepted do I notice I made an error.
``db_user.txt`` and ``db_pass.txt`` here are supposed to be ``root_db_user.txt`` and ``root_db_pass.txt``, respectively.